app/frontend/Calendar/DayView/WeekView.svelte

```diff
@@ -83,28 +83,24 @@
   }
 
   let days: Date[] = [];
+  let visibleEvents: Collection<Event>;
+  let allDayEvents: Collection<Event>;
   $: start, setDays();
   function setDays() {
     let startTime = new Date(start);
     if (showDays > 3) {
       startTime.setDate(startTime.getDate() - 1);
     }
-    startTime.setHours(startHour);
-    startTime.setMinutes(0);
-    startTime.setSeconds(0);
-    startTime.setMilliseconds(0);
+    let filterStart = startTime.setHours(startHour, 0, 0, 0);
```

Comment:
```suggestion
    startTime.setHours(startHour, 0, 0, 0);
    let filterStart = startTime.clone();
```

Do not combine 2 unrelated actions into the same line.
Readability over shortness.

clone(): See comment below